extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -32,55 +35,112 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
     /** @private {?Element} */
     this.element_ = element;
 
-    /** @private {?Element} */
-    this.image_ = null;
+    /** @private {?Element} The element that is transitioned. */
+    this.ampImgEl_ = dev().assertElement(
+      this.element_.querySelector('amp-img')
```

Comment:
Can we delay the querying of elements to the `buildCallback` and just initialize this as null?